src/WebUI/Pods/PodsDetails.tsx

```diff
@@ -161,6 +164,9 @@ export class PodsDetails extends BaseComponent<IPodsDetailsProperties, IPodsDeta
             selectedImageDetails: undefined
         };
     }
+    public componentDidMount(): void {
```

Comment:
keep this after render() and also have a new line between functions. applicable in all cases